.github/workflows/windows.yml

```
@@ -221,7 +227,7 @@ jobs:
           if [ -x "$(cygpath "${SYSTEMROOT}/System32/curl.exe")" ]; then
             TFLAGS+=" -ac $(cygpath "${SYSTEMROOT}/System32/curl.exe")"
           fi
```

```suggestion
          fi
          choco install --yes --no-progress --limit-output --timeout 180 stunnel --force
```
I suggest moving the `choco` command here, to simplify, and to not delay the build itself.
Also for the other two workflows.